CONTRIBUTING.md

```
@@ -1,78 +1,78 @@
+- `security`
+- `web`
+- `engines`
+
+**Good commit messages:**
+- Clear and descriptive
+- Start with lowercase (after type)
+- No period at the end
+- Imperative mood (add, fix, update, not adds, fixed, updating)
+
+## Project Structure
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)